packages/mdc-slider/README.md

```diff
@@ -40,15 +40,16 @@ npm install @material/slider
 ### HTML Structure
 
 ```html
-<div class="mdc-slider" tabindex="0" role="slider"
-     aria-valuemin="0" aria-valuemax="100" aria-valuenow="0"
-     aria-label="Select Value">
+<div class="mdc-slider" >
   <div class="mdc-slider__track">
     <div class="mdc-slider__track-fill"></div>
   </div>
-  <div class="mdc-slider__thumb">
-    <svg class="mdc-slider__thumb-handle" width="24" height="24">
-      <circle cx="12" cy="12" r="6"></circle>
+  <div class="mdc-slider__thumb" tabindex="0" role="slider"
+    aria-valuemin="0" aria-valuemax="50" aria-valuenow="20"
+    aria-label="Select Value">
+    <svg class="mdc-slider__thumb-handle" width="34" height="34">
+      <circle class="mdc-slider__thumb-handle-disabled" cx="17" cy="17" r="6"></circle>
+      <circle cx="17" cy="17" r="6"></circle>
```

Comment:
Is this element unnecessary since there's the disabled `<circle>` handle above?

Reply:
It is used to create a background of color "surface" with 100% opacity so the thumb does not have the intersection of the track shown.